hypha/core/navigation.py

```diff
@@ -0,0 +1,96 @@
+    COMMUNITY_REVIEWER_GROUP_NAME,
+    CONTRACTING_GROUP_NAME,
+    FINANCE_GROUP_NAME,
+    PARTNER_GROUP_NAME,
+    REVIEWER_GROUP_NAME,
+    STAFF_GROUP_NAME,
+    TEAMADMIN_GROUP_NAME,
+)
+
+nav_items = [
+    {
+        "title": "My Dashboard",
+        "url": "dashboard:dashboard",
+        "user_roles": [
+            STAFF_GROUP_NAME,
+            APPROVER_GROUP_NAME,
+            APPLICANT_GROUP_NAME,
+            REVIEWER_GROUP_NAME,
+            CONTRACTING_GROUP_NAME,
+            FINANCE_GROUP_NAME,
+            PARTNER_GROUP_NAME,
+            TEAMADMIN_GROUP_NAME,
+            COMMUNITY_REVIEWER_GROUP_NAME,
+        ],
+    },
+    {
+        "title": "Submissions",
+        "url": "apply:submissions:overview",  # kind of basic url to figure out active tab
+        "user_roles": [STAFF_GROUP_NAME, TEAMADMIN_GROUP_NAME],
+        "categories": [
+            {
+                "title": "All Submissions",
+                "url": "apply:submissions:list",
+                "user_roles": [STAFF_GROUP_NAME, TEAMADMIN_GROUP_NAME],
+            },
+            {
+                "title": "Staff Assignments",
+                "url": "apply:submissions:staff_assignments",
+                "user_roles": [STAFF_GROUP_NAME, TEAMADMIN_GROUP_NAME],
+            },
+            {
+                "title": "Reviews",
+                "url": "apply:submissions:reviewer_leaderboard",
+                "user_roles": [STAFF_GROUP_NAME, TEAMADMIN_GROUP_NAME],
+            },
+            {
+                "title": "Results",
+                "url": "apply:submissions:result",
+                "user_roles": [STAFF_GROUP_NAME, TEAMADMIN_GROUP_NAME],
+            },
+        ],
+    },
+    {
+        "title": "Projects",
+        "url": "apply:projects:overview",  # kind of basic url to figure out active tab
+        "user_roles": [
+            STAFF_GROUP_NAME,
+            TEAMADMIN_GROUP_NAME,
+            FINANCE_GROUP_NAME,
+            CONTRACTING_GROUP_NAME,
+        ],
+        "categories": [
+            {
+                "title": "All Projects",
+                "url": "apply:projects:all",
+                "user_roles": [
+                    STAFF_GROUP_NAME,
+                    TEAMADMIN_GROUP_NAME,
+                    FINANCE_GROUP_NAME,
+                    CONTRACTING_GROUP_NAME,
+                ],
+            },
+            {
+                "title": "Invoices",
+                "url": "apply:projects:invoices",
+                "user_roles": [
+                    STAFF_GROUP_NAME,
+                    TEAMADMIN_GROUP_NAME,
+                    FINANCE_GROUP_NAME,
+                ],
+            },
+            {
+                "title": "Reports",
+                "url": "apply:projects:reports:all",
+                "user_roles": [
+                    STAFF_GROUP_NAME,
+                    TEAMADMIN_GROUP_NAME,
+                    FINANCE_GROUP_NAME,
+                ],
+            },
+        ],
+    },
+]
```

Comment:
@frjo Now it can be easily managed via settings. We just have to pass a compact JSON in the env variable.
Like:   `APPLY_NAV_SUBMISSIONS_ITEMS='[{"title":"All Submissions","url":"apply:submissions:list","permission_method":"is_apply_staff_or_reviewer_required"},{"title":"Staff Assignments","url":"apply:submissions:staff_assignments","permission_method":"is_apply_staff"},{"title":"Reviews","url":"apply:submissions:reviewer_leaderboard","permission_method":"is_apply_staff"},{"title":"Results","url":"apply:submissions:result","permission_method":"is_apply_staff"}]'`